frontend/public/components/storage/create-pvc.tsx

```diff
@@ -38,7 +38,7 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
     requestSizeValue: '',
     requestSizeUnit: 'Gi',
     disableForm: false,
-    showLabelSelectors: false,
+    useSelector: false,
```

Comment:
Renamed this since it's not just controlling display. It affects the object that's created. (There might be a value set in state, then the checkbox unchecked.)